### ISSUE TITLE
Configure Name dialog should show CBOR diagnostic notation

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/electrum-nmc#316]
It would be nice if the Configure Name dialog showed the corresponding CBOR Diagnostic Notation when the value being entered is valid CBOR, so that the user can tell what their CBOR value will do.  Since CBOR is not part of the Python standard library, we should use Qt's CBOR functionality (Qt supports converting CBOR to Diagnostic Notation).

Blocked by https://github.com/namecoin/electrum-nmc/issues/314
